node_modules/oae-principals/lib/test/util.js

```diff
@@ -270,6 +270,18 @@ var assertGetUserFails = module.exports.assertGetUserFails = function(restCtx, u
 var assertGetMeSucceeds = module.exports.assertGetMeSucceeds = function(restCtx, callback) {
     RestAPI.User.getMe(restCtx, function(err, me, response) {
         assert.ok(!err);
+
+        // Sanity check the me model
+        assert.ok(me);
+        assert.ok(me.tenant);
+        assert.ok(_.isString(me.tenant.alias));
+        assert.ok(_.isString(me.tenant.displayName));
+        if (me.tenant.alias === 'guest') {
+            assert.strictEqual(me.tenant.isGuestTenant, true);
+        } else {
+            assert.ok(!_.chain(me.tenant).keys().contains('isGuestTenant').value());
```

Comment:
It ensures that the value isn't `undefined`, but in JS it's possible to have a key whose value is set to `undefined`.

Though I didn't consider though that it's coming from a JSON response, so it's probably fine to do `_.isUndefined`.